CMakeLists.txt

```diff
@@ -44,7 +44,8 @@ set(PLATFORM "PC_UNIX" CACHE STRING
         "Defines on what platform the application will run")
 
 option(INSTALL_LOCALLY "Install this project locally" OFF)
-option(BUILD_TESTS "Build tests (requires gtest)" OFF)
+option(BUILD_TESTS "Build framework tests (requires gtest)" OFF)
+option(BUILD_CRYPTOGRAPHY_TESTS "Build cryptography tests" ON)
```

Comment:
Why by default on ? If I do not pass it now, it will be build. I think the majority off the people are not interested in this :-)